grails-app/conf/BootStrap.groovy

```diff
@@ -249,7 +249,20 @@ class BootStrap {
 
 
         JSON.registerObjectMarshaller(Product) { Product product ->
-            return product.toJson()
+            [
```

Comment:
Why are we moving these out of the Product class? I am trying to move all of these into the domain classes.